jdxsmith/whats-cookin
# Feature/create class structure

#### What does this PR do?
 - Creates the class structure (`User.js, Recipe.js, Pantry.js`) along with the corresponding tests

#### Where should the reviewer start?
 - Start in the User.js file, then move to Recipe.js, then move to Pantry.js
 - Check each corresponding test file

#### How should this be manually tested?
 - Run each test file using `npm test` to ensure all tests pass
 - Look through tests to make sure correct functionality is being tested

#### Any background context you want to provide?
 - These classes will be used in the scripts.js file when the DOM is incorporated
